addons/mail/static/src/discuss/call/common/rtc_service.js

```diff
@@ -1676,6 +1676,23 @@ export class Rtc extends Record {
         }
     }
 
+    /**
+     * Applies blur effect to a video stream using BlurManager.
+     *
+     * @param {MediaStream} videoStream - input video stream.
+     * @returns {Promise<{stream: MediaStream, close: Function}>} - Blurred video stream and a function to close the blur manager.
+     */
+    async applyBlurEffect(videoStream) {
+        const blurManager = new BlurManager(videoStream, {
+            backgroundBlur: this.store.settings.backgroundBlurAmount,
+            edgeBlur: this.store.settings.edgeBlurAmount,
+        });
+        return {
```

Comment:
Is there a reason it's returning an object instead of the `BlurManger` instance? We need to be able to access the instance so that we can modify its public properties `edgeBlur` and `backgroundBlur` to update the blur properties when it's running.

see: https://github.com/odoo/odoo/blob/19.0/addons/mail/static/src/discuss/call/common/rtc_service.js#L420-L425